### PR TITLE
Fixing the asking of credentials in Iceberg.

### PR DESCRIPTION
**Pull request by tesonep**
The initialization of the dialog should be done in #initializeDialogWindow:

### CODE REVIEW

**Comment by tesonep** [comment]
Fix #1418

**Comment by tesonep** [comment]
Fix #1410 also

**Comment by tinchodias** [comment]
Looks good, great. I will try it.

**Comment by astares** [comment]
Maybe that is also solving https://github.com/pharo-vcs/iceberg/issues/1410

**Review by tinchodias — APPROVED** [review]

**Comment by tinchodias** [comment]
I'm working on IceTipAskForPlaintextCredentialsModelTest, whose tests are broken with this change

File: Iceberg-TipUI/IceTipDialog.class.st [hunk 1]
```diff
@@ -20,11 +20,19 @@ IceTipDialog >> cancel [
...
 		  okAction: [ self accept ];
-		  cancelAction: [ self cancel ];
-		  yourself
+		  cancelAction: [ self cancel ].
+]
+
+{ #category : #showing }
+IceTipDialog >> openDialogWithSpec [
+
+	^ self openModalWithSpec
```
Comment (tinchodias) [inline Iceberg-TipUI/IceTipDialog.class.st:36]: @tesonep Tests started to fail because of this change. I will create issue to explain it there.

Comment (tinchodias) [reply Iceberg-TipUI/IceTipDialog.class.st:36]: #1438

**Pull request merged**